### ISSUE TITLE
Coroutines BLE API

### ISSUE THREAD

**Issue by yschimke** [garanj/wearwind#17]
Just curious, have you looked at https://github.com/Beepiz/BleGattCoroutines ?

**Comment by garanj** [comment]
No, looks good! Would be great to overhaul this app.

**Comment by yschimke** [comment]
Yeah, also wonder if there are ways to achieve this for other devices. Like via Home Assistant APIs. But not sure anyone is asking for that :)
